Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -23,7 +23,9 @@ public static LootLockerConfig Get()
             if (settingsInstance != null)
             {
                 settingsInstance.ConstructUrls();
-                settingsInstance?.CheckForSettingOverrides();
+#if LOOTLOCKER_COMMANDLINE_SETTINGS
+            settingsInstance?.CheckForSettingOverrides();
```

Comment:
This question-mark is not needed, since you null-check in the if-statement above.

Reply:
I mean, if you don't feel like changing this, that's fine, this is not anything that will go towards our users really, and the added overhead from this is very very small, so, I will approve this anyway.